code/modules/mob/living/carbon/racial_spells.dm

```diff
@@ -0,0 +1,35 @@
+//For spells designed to be inherent abilities given to mobs via their species datum
+
+/spell/racial
```

Comment:
I figured it'd be better than setting the `panel` and `override_base` vars on each new racial spell, but I can remove the subtype if you think that would be better.